proto/provenance/metadata/v1/tx.proto

```diff
@@ -107,6 +110,11 @@ message MsgWriteScopeRequest {
   // If there is a value in scope.specification_id that is different from the one created from this uuid, an error is
   // returned.
   string spec_uuid = 4 [(gogoproto.moretags) = "yaml:\"spec_uuid\""];
+
+  // usd_mills value of scope in usd mills (1234 = $1.234) used for net asset value
+  uint64 usd_mills = 12;
+  // volume value of scope used for net asset value
+  uint64 volume = 13;
```

Comment:
Not sure we need this parameter because it should always be one as a non-fungible token.

Reply:
Good point.  I will remove this and all associated code along with it.